ThreeBotPackages/tfgrid_solutions/tfgrid_solutions/chatflows/your_flist.py

```diff
@@ -155,6 +198,10 @@ def container_pay(self):
             cpu=self.user_form_data["CPU"],
             memory=self.user_form_data["Memory"],
         )
+        if self.container_volume_attach:
+            j.sal.zosv2.volume.attach_existing(
```

Comment:
so if you do't wonna make an extra reservation for the volume itself you instead of this method need to use

j.sal.zosv2.volume.attach(container, volume, mount_point)

Reply:
thanks

Reply:
Combined reservations in one : https://github.com/threefoldtech/jumpscaleX_threebot/pull/746/commits/8f141505b14c31471750da6d21ce95267d67b2b5